### PR TITLE
qm-sendkey: add page

### PR DESCRIPTION
**Pull request by deeeeeeps**
Requested here - #8537

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/qm-sendkey.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# qm sendkey
+
+> Send the Qemu monitor encoding key event to the virtual machine.
```
Comment (kbdharun) [inline pages/linux/qm-sendkey.md:3]:
```suggestion
> Send the QEMU monitor encoding key event to the virtual machine.
```

File: pages/linux/qm-sendkey.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Send the specified key event to the virtual machine:
```
Comment (kbdharun) [inline pages/linux/qm-sendkey.md:6]:
```suggestion
- Send the specified key event to a specific virtual machine:
```

**Comment by deeeeeeps** [comment]
Thank you @kbdharun

File: pages/linux/qm-sendkey.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# qm sendkey
+
+> Send the QEMU monitor encoding key event to the virtual machine.
```
Comment (kbdharun) [inline pages/linux/qm-sendkey.md:3]:
```suggestion
> Send the QEMU monitor encoding key event to a specific virtual machine.
```
I guess the same can be applied here too.

File: pages/linux/qm-sendkey.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# qm sendkey
+
+> Send the QEMU monitor encoding key event to a specific virtual machine.
```
Comment (kbdharun) [inline pages/linux/qm-sendkey.md:3]:
```suggestion
> Send  QEMU monitor encoding key event to a specific virtual machine.
```
I updated the suggestion seems like you committed it fast 😄

Comment (deeeeeeps) [reply pages/linux/qm-sendkey.md:3]: Ah I was wondering what happened. I will wait for few more minutes before I commit 😅

Comment (kbdharun) [reply pages/linux/qm-sendkey.md:3]: > Ah I was wondering what happened. I will wait for few more minutes before I commit sweat_smile

I guess you can commit it now.

**Review by kbdharun — APPROVED** [review]

File: pages/linux/qm-sendkey.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# qm sendkey
+
+> Send QEMU monitor encoding key event to a specific virtual machine.
```
Comment (kbdharun) [inline pages/linux/qm-sendkey.md:3]:
```suggestion
> Send QEMU monitor encoding key event to a virtual machine.
```
I think we don't need to add `specific` keyword to the page description as it is already present in the command description. What do you think?

Comment (deeeeeeps) [reply pages/linux/qm-sendkey.md:3]: Sounds good

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**